content/matplotlib/concepts/pyplot/terms/subplots/subplots.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: '.subplots()'
+Description: 'Adds a given number of subplots to a matplotlib figure.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'visualize-data-with-python'
+---
+
+The **`.subplots()`** method returns a figure with multiple subplots. This method is an efficient way to create popular subplot layouts with a high-level and concise syntax.
+
+## Syntax
+
+```pseudo
+pyplot.subplots(nrows, ncols)
+```
+
+`.subplots()` has the following parameters:
+
+|  Parameter Name  | Data Type(s)                                                        | Usage                                                                                                                                                                                                                                |
+| :--------------: | :------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|  `nrows, ncols`  | int                                                                 | The quantity of rows and/or columns within the grid of subplots. Defaults to 1.                                                                                                                                                      |
+| `sharex, sharey` | bool or one of the following arguments: 'none', 'all', 'row', 'col' | `sharex` and `sharey` control the sharing of properties among x or y axis of subplots. Shared x-axis or y-axis allows for creating tick labels only on the bottom subplot or first column subplot, respectively. Default is `False`. |
+|    `squeeze`     | bool                                                                | Removes additional dimensions from the array of Axes when set to `True` (default). If set to `False`, it will always return a 2D array of Axes instances.                                                                            |
```

Comment:
```suggestion
|    `squeeze`     | bool                                                                | Removes additional dimensions from the array of `Axes` when set to `True` (default). If set to `False`, it will always return a 2D array of `Axes` instances.                                                                            |
```